### ISSUE TITLE
Running .\build.ps1 fails

### ISSUE THREAD

**Issue by DonSchenck** [openstacknetsdk/openstack.net#489]
Clean Windows 8.1 machine with Visual Studio 2015 CTP 6:
```
_CopyAppConfigFile:
  Copying file from "app.config" to "bin\Release\OpenStackNetTests.Unit.dll.config".
CopyFilesToOutputDirectory:
  Copying file from "obj\Release\OpenStackNetTests.Unit.dll" to "bin\Release\OpenStackNetTests.Unit.dll".
  OpenStackNetTests.Unit.net45 -> C:\Users\User\Documents\GitHub\openstack.net\src\OpenStackNetTests.Unit\bin\Release\O
  penStackNetTests.Unit.dll
  Copying file from "obj\Release\OpenStackNetTests.Unit.pdb" to "bin\Release\OpenStackNetTests.Unit.pdb".
Done Building Project "C:\Users\User\Documents\GitHub\openstack.net\src\OpenStackNetTests.Unit\OpenStackNetTests.Unit.n
et45.csproj" (Rebuild target(s)).

Done Building Project "C:\Users\User\Documents\GitHub\openstack.net\src\openstack.net.sln" (rebuild target(s)) -- FAILE
D.


Build FAILED.

"C:\Users\User\Documents\GitHub\openstack.net\src\openstack.net.sln" (rebuild target) (1) ->
"C:\Users\User\Documents\GitHub\openstack.net\src\OpenStack.Net\OpenStack.Net.net35.csproj" (Rebuild target) (10) ->
(CoreCompile target) ->
  OpenStack\Net\IPAddress.cs(38,24): warning CS1574: XML comment on 'OpenStack.Net.IPAddress._address4' has cref attrib
ute 'AddressFamily.Ipv4' that could not be resolved [C:\Users\User\Documents\GitHub\openstack.net\src\OpenStack.Net\Ope
nStack.Net.net35.csproj]
  OpenStack\Net\IPAddress.cs(44,24): warning CS1574: XML comment on 'OpenStack.Net.IPAddress._address6' has cref attrib
ute 'AddressFamily.Ipv6' that could not be resolved [C:\Users\User\Documents\GitHub\openstack.net\src\OpenStack.Net\Ope
nStack.Net.net35.csproj]


"C:\Users\User\Documents\GitHub\openstack.net\src\openstack.net.sln" (rebuild target) (1) ->
"C:\Users\User\Documents\GitHub\openstack.net\src\OpenStack.Net\OpenStack.Net.net40.csproj" (Rebuild target) (11) ->
  OpenStack\Net\IPAddress.cs(38,24): warning CS1574: XML comment on 'OpenStack.Net.IPAddress._address4' has cref attrib
ute 'AddressFamily.Ipv4' that could not be resolved [C:\Users\User\Documents\GitHub\openstack.net\src\OpenStack.Net\Ope
nStack.Net.net40.csproj]
  OpenStack\Net\IPAddress.cs(44,24): warning CS1574: XML comment on 'OpenStack.Net.IPAddress._address6' has cref attrib
ute 'AddressFamily.Ipv6' that could not be resolved [C:\Users\User\Documents\GitHub\openstack.net\src\OpenStack.Net\Ope
nStack.Net.net40.csproj]


"C:\Users\User\Documents\GitHub\openstack.net\src\openstack.net.sln" (rebuild target) (1) ->
"C:\Users\User\Documents\GitHub\openstack.net\src\OpenStack.Net\OpenStack.Net.net45.csproj" (Rebuild target) (12) ->
  OpenStack\Net\IPAddress.cs(38,24): warning CS1574: XML comment on 'OpenStack.Net.IPAddress._address4' has cref attrib
ute 'AddressFamily.Ipv4' that could not be resolved [C:\Users\User\Documents\GitHub\openstack.net\src\OpenStack.Net\Ope
nStack.Net.net45.csproj]
  OpenStack\Net\IPAddress.cs(44,24): warning CS1574: XML comment on 'OpenStack.Net.IPAddress._address6' has cref attrib
ute 'AddressFamily.Ipv6' that could not be resolved [C:\Users\User\Documents\GitHub\openstack.net\src\OpenStack.Net\Ope
nStack.Net.net45.csproj]


"C:\Users\User\Documents\GitHub\openstack.net\src\openstack.net.sln" (rebuild target) (1) ->
"C:\Users\User\Documents\GitHub\openstack.net\src\OpenStack.Net\OpenStack.Net.portable-net40.csproj" (Rebuild target) (
14) ->
  OpenStack\Net\IPAddress.cs(38,24): warning CS1574: XML comment on 'OpenStack.Net.IPAddress._address4' has cref attrib
ute 'AddressFamily.Ipv4' that could not be resolved [C:\Users\User\Documents\GitHub\openstack.net\src\OpenStack.Net\Ope
nStack.Net.portable-net40.csproj]
  OpenStack\Net\IPAddress.cs(44,24): warning CS1574: XML comment on 'OpenStack.Net.IPAddress._address6' has cref attrib
ute 'AddressFamily.Ipv6' that could not be resolved [C:\Users\User\Documents\GitHub\openstack.net\src\OpenStack.Net\Ope
nStack.Net.portable-net40.csproj]


"C:\Users\User\Documents\GitHub\openstack.net\src\openstack.net.sln" (rebuild target) (1) ->
"C:\Users\User\Documents\GitHub\openstack.net\src\OpenStack.Net\OpenStack.Net.portable-net45.csproj" (Rebuild target) (
15) ->
  OpenStack\Net\IPAddress.cs(38,24): warning CS1574: XML comment on 'OpenStack.Net.IPAddress._address4' has cref attrib
ute 'AddressFamily.Ipv4' that could not be resolved [C:\Users\User\Documents\GitHub\openstack.net\src\OpenStack.Net\Ope
nStack.Net.portable-net45.csproj]
  OpenStack\Net\IPAddress.cs(44,24): warning CS1574: XML comment on 'OpenStack.Net.IPAddress._address6' has cref attrib
ute 'AddressFamily.Ipv6' that could not be resolved [C:\Users\User\Documents\GitHub\openstack.net\src\OpenStack.Net\Ope
nStack.Net.portable-net45.csproj]


"C:\Users\User\Documents\GitHub\openstack.net\src\openstack.net.sln" (rebuild target) (1) ->
"C:\Users\User\Documents\GitHub\openstack.net\src\Samples\CPPCodeSamples\CPPCodeSamples.vcxproj" (Rebuild target) (8) -
>
(PlatformPrepareForBuild target) ->
  C:\Program Files (x86)\MSBuild\Microsoft.Cpp\v4.0\V120\Microsoft.Cpp.Platform.targets(64,5): error MSB8020: The build
 tools for v140 (Platform Toolset = 'v140') cannot be found. To build using the v140 build tools, please install v140 b
uild tools.  Alternatively, you may upgrade to the current Visual Studio tools by selecting the Project menu or right-c
lick the solution, and then selecting "Upgrade Solution...". [C:\Users\User\Documents\GitHub\openstack.net\src\Samples\
CPPCodeSamples\CPPCodeSamples.vcxproj]


"C:\Users\User\Documents\GitHub\openstack.net\src\openstack.net.sln" (rebuild target) (1) ->
"C:\Users\User\Documents\GitHub\openstack.net\src\Samples\FSharpCodeSamples\FSharpCodeSamples.fsproj" (Rebuild target)
(9) ->
  C:\Users\User\Documents\GitHub\openstack.net\src\Samples\FSharpCodeSamples\FSharpCodeSamples.fsproj(81,3): error MSB4
102: The value "" of the "Project" attribute in element <Import> is invalid. Parameter "path" cannot have zero length.


"C:\Users\User\Documents\GitHub\openstack.net\src\openstack.net.sln" (rebuild target) (1) ->
"C:\Users\User\Documents\GitHub\openstack.net\src\OpenStack.Net\OpenStack.Net.netcore45.csproj" (Rebuild target) (13) -
>
  C:\Users\User\Documents\GitHub\openstack.net\src\OpenStack.Net\OpenStack.Net.netcore45.csproj(205,3): error MSB4019:
The imported project "C:\Program Files (x86)\MSBuild\Microsoft\WindowsXaml\v12.0\Microsoft.Windows.UI.Xaml.CSharp.targe
ts" was not found. Confirm that the path in the <Import> declaration is correct, and that the file exists on disk.

    10 Warning(s)
    3 Error(s)

Time Elapsed 00:00:15.84
Build failed, aborting!
```

**Comment by sharwell** [comment]
The default build is configured to run with the "12.0" toolset, which is standard in Visual Studio 2013. To build with the new tools, use the following command:

```powershell
.\build.ps1 -VisualStudioVersion 14.0
```

If the SHFB tools are not installed on the machine, you may need to also add the `-InstallSHFB` flag to that command line.

**Comment by DonSchenck** [comment]
New box, installed VS 2013 Community ... getting a TON of these when running build.ps1:
```
"C:\Users\Administrator\Documents\Visual Studio 2013\Projects\openstack.net\src\openstack.net.sln" (rebuild targ
et) (1) ->
"C:\Users\Administrator\Documents\Visual Studio 2013\Projects\openstack.net\src\OpenStack.Net\OpenStack.Net.netc
ore45.csproj" (Rebuild target) (14) ->
  C:\Users\Administrator\Documents\Visual Studio 2013\Projects\openstack.net\src\packages\Rackspace.KeyReporting
.1.0.0\build\Rackspace.KeyReporting.targets(14,5): error : This project references a strong name key that is mis
sing on this computer. Run 'sn -k C:\Users\Administrator\Documents\Visual Studio 2013\Projects\keys\openstack.ne
t.netcore45.snk' to generate the file. [C:\Users\Administrator\Documents\Visual Studio 2013\Projects\openstack.n
et\src\OpenStack.Net\OpenStack.Net.netcore45.csproj]


"C:\Users\Administrator\Documents\Visual Studio 2013\Projects\openstack.net\src\openstack.net.sln" (rebuild targ
et) (1) ->
"C:\Users\Administrator\Documents\Visual Studio 2013\Projects\openstack.net\src\OpenStack.Net\OpenStack.Net.port
able-net40.csproj" (Rebuild target) (15) ->
  C:\Users\Administrator\Documents\Visual Studio 2013\Projects\openstack.net\src\packages\Rackspace.KeyReporting
.1.0.0\build\Rackspace.KeyReporting.targets(14,5): error : This project references a strong name key that is mis
sing on this computer. Run 'sn -k C:\Users\Administrator\Documents\Visual Studio 2013\Projects\keys\openstack.ne
t.portable-net40.snk' to generate the file. [C:\Users\Administrator\Documents\Visual Studio 2013\Projects\openst
ack.net\src\OpenStack.Net\OpenStack.Net.portable-net40.csproj]


"C:\Users\Administrator\Documents\Visual Studio 2013\Projects\openstack.net\src\openstack.net.sln" (rebuild targ
et) (1) ->
"C:\Users\Administrator\Documents\Visual Studio 2013\Projects\openstack.net\src\OpenStack.Net\OpenStack.Net.port
able-net45.csproj" (Rebuild target) (16) ->
  C:\Users\Administrator\Documents\Visual Studio 2013\Projects\openstack.net\src\packages\Rackspace.KeyReporting
.1.0.0\build\Rackspace.KeyReporting.targets(14,5): error : This project references a strong name key that is mis
sing on this computer. Run 'sn -k C:\Users\Administrator\Documents\Visual Studio 2013\Projects\keys\openstack.ne
t.portable-net45.snk' to generate the file. [C:\Users\Administrator\Documents\Visual Studio 2013\Projects\openst
ack.net\src\OpenStack.Net\OpenStack.Net.portable-net45.csproj]
```

**Comment by sharwell** [comment]
This was fixed in #476. If you are working on a branch that does not include that commit, you can run the commands indicated in those error messages to create the necessary files. This intermediate step of generating keys for the build will disappear once the code is merged to include the latest work in the `master` branch.

An example of one of these commands from your output (I added the quotes because they are required due to the space in the path):

```
sn -k "C:\Users\Administrator\Documents\Visual Studio 2013\Projects\keys\openstack.ne
t.portable-net45.snk"
```

**Comment by carolynvs** [comment]
Fixed by #513
